.travis.yml

```diff
@@ -8,3 +8,7 @@ install:
 script:
     - nix-shell --run 'cabal test'
     - nix-shell --run '$(dirname $NIX_GHC)/runhaskell -package-db=$NIX_GHC_LIBDIR/package.conf.d test/typecheck/Spec.hs'
+    - cat dist/test/biegunka-0.2-spec.log
```

Comment:
I was curious about what happens in tests when there are not failing. Since `updateGit` is tested now instead of `biegunka` calling `git`, we don't need it no more.